community/CM-Configuration-Management/README.ACM

```diff
@@ -0,0 +1,46 @@
+Steps to Deploy ZTS-CMC-APP
```

Comment:
Is this README just your notes about setting up ZTS and not something you intend to include?  I don't think we want to have an ACM readme in this directory that is only for ZTS.  Can we include any notes in the policy and the README.md file or include links to separate documentation?

Reply:
I removed this README and added a link to github for zettset.